compiler/ilgen/IlBuilder.hpp

```diff
@@ -142,6 +142,8 @@ class IlBuilder : public TR::IlInjector
    // create a new local value (temporary variable)
    TR::IlValue *NewValue(TR::IlType *dt);
 
+   TR::IlValue *CopyValue(TR::IlValue *value);
```

Comment:
given the API, I would be tempted to call it "Copy" rather than "CopyValue" .... I know it's kinda like NewValue, but I think of it more like Load, Store, Add, Sub which also could (but by choice did not) get a Value suffix.

I could be argued out of it, but just think of all the characters you will no longer have to type :) .